docs/docs/resources/pipeline-components/kafka-source-connector.yaml

```diff
@@ -40,4 +39,5 @@
   # Image Tag etc.
   resetterValues:
     imageTag: 1.2.3
+  #  https://kafka.apache.org/documentation/#impl_offsettracking
```

Comment:
That is actually the wrong link. This refers to the standard Kafka Consumer group offsets. However, here we refer to Kafka Connect offsets for source connectors, which is quite a bit different

Reply:
Is it [offset.storage.topic](https://kafka.apache.org/documentation/#connect_running) or [offset.storage.file.filename](https://kafka.apache.org/documentation/#connect_running)?